Core/src/EventData/PrintParameters.cpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/EventData/detail/PrintParameters.hpp"
+
+#include <ostream>
+
+#include "Acts/Surfaces/Surface.hpp"
+
+void Acts::detail::printBoundParameters(std::ostream& os,
+                                        const Acts::Surface& surface,
+                                        const Acts::BoundVector& params,
+                                        const Acts::BoundSymMatrix* cov) {
+  // Set stream output format
+  auto oldPrecision = os.precision(7);
+  auto oldFlags = os.setf(std::ios::fixed);
+
+  os << "BoundTrackParameters:\n";
+  os << "  parameters: " << params.transpose() << '\n';
+  if (cov) {
+    os << "  covariance:\n";
+    os << *cov << '\n';
+  } else {
+    os << "  no covariance stored";
+  }
+  os << "  on surface:";
+  surface.toStream(GeometryContext(), os);
```

Comment:
The `toStream` prints also the position and orientation of the surface. I would argue against a default `GeometryContext` here.

Reply:
What should we use instead? Maybe just the surface name?

Reply:
I would suggest that either type+geo id (less verbose, more complicated for debugging) or passing the geometry context to this function (more verbose, easier debugging).

Reply:
I do not want add the geometry id here, since the whole point of the bound parameters simplification is to remove it. I will use the first option, then.

Reply:
Updated as discussed.